src/sentry/static/sentry/app/components/avatar/index.tsx

```diff
@@ -6,14 +6,15 @@ import OrganizationAvatar from 'app/components/avatar/organizationAvatar';
 import ProjectAvatar from 'app/components/avatar/projectAvatar';
 import TeamAvatar from 'app/components/avatar/teamAvatar';
 import UserAvatar from 'app/components/avatar/userAvatar';
-import {Team, OrganizationSummary, AvatarProject} from 'app/types';
+import {Team, OrganizationSummary, AvatarProject, Member} from 'app/types';
 
 const BasicModelShape = PropTypes.shape({slug: PropTypes.string});
 
 type Props = {
   team?: Team;
   organization?: OrganizationSummary;
   project?: AvatarProject;
+  member?: Member;
```

Comment:
`member` is unused at this point. I think you need to add `member` into the render function?

Reply:
This is not necessary anymore...I remove it 😉